vuln-remediation.md

```diff
@@ -0,0 +1,105 @@
+
+   1. **Coordinator identifies and notifies the responsible Maintainer.** When a vulnerability is in scope, follow "additional guidance" below on how to identify the Maintainer. If secure communications have been established with the Reporter, secure communications should also be set up with the Maintainer. Use the following template to reach out to the Maintainer, include a copy of the message to [security@openrobotics.org](mailto:security@openrobotics.org) (the template may be modified as needed to fit the situation):
+
+      > SUBJECT: [HIGH | Medium | Low] risk security vulnerability in [package name]
+      > 
+      > A [HIGH | Medium | Low] risk security vulnerability has been identified in [package name].  Please contact me at [contact] to discuss the vulnerability details.
+      > 
+      > [include if needed] Based on the severity of the vulnerability, the details are particularly sensitive and need to be protected through secure communications. Please send me your public PGP key through which you can receive secure email; if you are unable to send and receive secure email, please let me know and we will establish another secure channel.
+      > 
+      > [Attach a PGP key or a Key ID]
+      >
+      > [Sign the email with the PGP key]
+      >
+
+   1. **Coordinator registers a CVE.** Seek help from the [ROS 2 security working group](https://github.com/ros-security/community#communication-channels) if necessary to reserve the CVE. The reserved CVE should not include any detailed information; this will be added after disclosure. All subsequent communications should include the CVE numnber for traceability.
+
+1. **Remediation**
+
+   1. **Maintainer fixes the vulnerability.** Work should be done locally as much as possible; when the fix is pushed to github the fix becomes visible to the public even though the patch has not been released to users. The final pull request for the fix should include a reference to the CVE in comments for traceability.
+
+   1. **Coordinator tracks to completion.**  This likely will mean periodic update requests to the Maintainer until the Maintainer publishes a fix. Seek assistance from the Security WG if needed.
+
+   1. **Coordinator plans for vulnerability disclosure.** For high risk vulnerabilities or fixes with significant impact, create a communications plan for patch release and full disclosure. The plan should take input from the Maintainer and the Reporter.
+
+1. **Disclosure**
+
+   1. **Maintainer conducts patch pre-notification.** When the fix is ready for release, the Maintainer, at his or her discretion, will notify select individuals that the fix is ready. This notification should be specifically to quickly remediate any build issues caused by the patch, and to finally merge the pull request.
+
+   1. **Coordinator notifies the Reporter that the patch has been released.**
```

Comment:
Clarification: after the previous step the fix has been MERGED. It still needs to be bloomed and even then won't be contained in the ROS distro until the next sync. Should we say something about that here?

Reply:
Addressed with b0aa799